application/views/bestitamazonpay4oxid_paybutton.tpl

```diff
@@ -0,0 +1,130 @@
+                new OffAmazonPayments.Widgets.Button ({
+                    sellerId: '[{$oViewConf->getAmazonConfigValue('sAmazonSellerId')}]',
+                    onSignIn: function(orderReference) {
+                        amazonOrderReferenceId = orderReference.getAmazonOrderReferenceId();
+                        [{if $addToCart}]
+                            window.location = '[{$oViewConf->getSslSelfLink()|html_entity_decode}]'
+                                + 'bestitAmazonPayIsAmazonPay=1&amazonOrderReferenceId=' + amazonOrderReferenceId
+                                + '&' + $("#payWithAmazonDiv").parents('form:first').serialize();
+                        [{else}]
+                            window.location = '[{$oViewConf->getSslSelfLink()|html_entity_decode}]'
+                                + 'cl=user&amazonOrderReferenceId=' + amazonOrderReferenceId;
+                        [{/if}]
+                    },
+                    onError: function(error) {
+                        window.location = '[{$oViewConf->getSslSelfLink()|html_entity_decode}]cl=basket&amazonOrderReferenceId=' + amazonOrderReferenceId;
+                    }
+                }).bind("payWithAmazonButton");
+            });
+        [{/capture}]
+        [{oxscript add=$smarty.capture.sBestitAmazonScript}]
+    [{/if}]
+
+    [{*Newer Amazon Pay & Login functionality !*}]
+    [{if $oViewConf->showAmazonPayButton()}]
+        [{assign var="sAmazonWidgetUrl" value=$oViewConf->getAmazonProperty('sAmazonLoginWidgetUrl')}]
+        [{assign var="sAmazonSellerId" value=$oViewConf->getAmazonConfigValue('sAmazonSellerId')}]
+        [{assign var="sModuleUrl" value=$oViewConf->getModuleUrl('bestitamazonpay4oxid')}]
+
+        [{oxscript include="`$sAmazonWidgetUrl`?sellerId=`$sAmazonSellerId`" priority=11}]
+        [{oxscript include="`$sModuleUrl`out/src/js/bestitamazonpay4oxid.js" priority=11}]
+        [{oxstyle  include="`$sModuleUrl`out/src/css/bestitamazonpay4oxid.css"}]
+
+        <div id="payWithAmazonDiv" class="amazonContentGroup">
+            <div class="amazonTooltip">
+                <i>?</i>
+                <div class="amazonTooltipContent">[{oxmultilang ident="BESTITAMAZONPAY_PAY_BUTTON_HINT"}]</div>
+            </div>
+            <div id="payWithAmazonButton"></div>
+        </div>
+
+
+        [{capture name="sBestitAmazonLoginScript"}]
+            $(document).ready(function () {
+                amazon.Login.setClientId('[{$oViewConf->getAmazonConfigValue('sAmazonLoginClientId')}]');
+
+                [{assign var="aButtonStyle" value="-"|explode:$oViewConf->getAmazonConfigValue('sAmazonPayButtonStyle')}]
+
+                var authRequest;
+                OffAmazonPayments.Button('payWithAmazonButton', '[{$oViewConf->getAmazonConfigValue('sAmazonSellerId')}]', {
+                    type: '[{$aButtonStyle.0}]',
+                    size: ($('meta[name=apple-mobile-web-app-capable]').attr("content")=='yes') ? 'medium' : 'small',
```

Comment:
You switch really often between single and double quotes. Here:
`$('meta[name=apple-mobile-web-app-capable]').attr("content")`

Reply:
Same here: The code of the button was just moved to an other file so I didn't any refactoring.